content/css/concepts/cursor/cursor.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'cursor'
+Description: 'Sets the appearance of the mouse cursor when hovering over an element.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Style'
+  - 'UI'
+CatalogContent:
+  - 'learn-css'
+  - 'learn-intermediate-css'
+---
+
+The `cursor` property sets the appearance of the mouse cursor when hovering over an element. The appearance can be set by specifying the type of cursor, or by loading a specific icon. When using an icon, fallback images can be used, ending with a mandatory keyword fallback.
+
+These are some common keywords used to specify the type of cursor to display:
+
+![Cursors](https://raw.githubusercontent.com/Codecademy/docs/main/media/cursors-1.png)
+
+## Syntax
+
+Without a URL:
+
+```pseudo
+cursor: keyword;
+```
+
+With one or more URLs:
+
+```pseudo
+cursor: url(cursor_image),url(fallback_cursor_image), keyword;
+```
+
+Where `cursor_image` is the URL of the primary image for the cursor, `fallback_cursor_image` is optional. It is the URL to load if the prior image fails. There can be any number of fallback images. `keyword` is a mandatory cursor specifier in case all the image URLs fail to load.
```

Comment:
```suggestion
If the primary `cursor_image` fails to load, an optional `fallback_cursor_image` can be applied instead. However, in case all the image URLs fail to load, the mandatory `keyword` will be used as a last resort.
```